tests/support/server.tcl

```diff
@@ -202,6 +202,11 @@ proc tags_acceptable {tags err_return} {
         return 0
     }
 
+    if {$::tls && [lsearch $tags "tls:skip"] >= 0} {
```

Comment:
I suppose we should document that one in the Readme.

Reply:
:+1: